src/main/java/br/ufes/inf/nemo/frameweb/vp/controllers/AddFrameWebDependencyToClassContextController.java

```diff
@@ -24,11 +38,38 @@
  * @version 0.0.1
  */
 public class AddFrameWebDependencyToClassContextController implements VPContextActionController {
+    /** The contents of the Add FrameWeb Dependency dialog. */
+    private SelectDAOPanel selectDAOPanel;
+
+    /** The Plug-in Settings dialog. */
+    private IDialog selectDAODialog;
+
+    /** A map of the DAO Interfaces of the project. */
+    private Map<String, IClass> daoClassesMap;
+
+    public Map<String, IClass> getDaoClassesMap(){
+        return this.daoClassesMap;
+    }
+
+    private VPContext context;
+
+    /** List of all packages present in the current diagram */
+    private List<IModelElement> currentDiagramPackages;
+    /** List of all DAO Interfaces present in the current diagram */
+    private List<IDiagramElement> currentDiagramDAOInterfaces;
+    /** List of all DAO Interfaces that have an association with the service class */
+    private List<IModelElement> associatedDAOInterfaces;
+    /** Application packages in the diagram*/
+    private List<IDiagramElement> applicationPackages;
+
+
     /** Called when the button is pressed. Adds the dependency to the class and diagram. */
     @Override
     public void performAction(final VPAction action, final VPContext context, final ActionEvent event) {
+        Logger.log(Level.CONFIG, "Performing action: Add FrameWeb Constraint (Class Attribute) > {0}",
```

Comment:
```suggestion
        Logger.log(Level.CONFIG, "Performing action: Add FrameWeb Dependency (Class) > {0}",
```

Aren't we adding a FrameWeb depedency to a class? Why Add FrameWeb Constraint (Class Attribute)? Am I missing something?

Reply:
My mistake. I copied this line from another class and forgot to change.